shell/edit/logging-flow/index.vue

```diff
@@ -312,6 +366,20 @@ export default {
       if (this.value.spec.match && this.isMatchEmpty(this.value.spec.match)) {
         delete this.value.spec['match'];
       }
+
+      if (this.loggingType === FLOW_AUDIT) {
+        this.$set(this.value.spec, 'loggingRef', 'harvester-kube-audit-log-ref');
```

Comment:
The `$set()` API has bee removed in Vue3, we should be able to use assignment to produce the same effect

https://v3-migration.vuejs.org/breaking-changes/#removed-apis

Reply:
Fixed